pubsub/google/cloud/pubsub_v1/subscriber/_consumer.py

```diff
@@ -334,14 +347,14 @@ def _blocking_consume(self, policy):
                 response_generator, self._can_consume)
             try:
                 for response in responses:
-                    _LOGGER.debug('Received response:\n%r', response)
```

Comment:
I did actually need these for debugging at a previous point. Perhaps still send them with a envvar or something? (Not critically important, no need to block the PR on them.)

Reply:
This can cause a large amount of CPU to be consumed as Python converts the response to a string every time regardless of if logging is turned on.

Reply:
Ack.